frontend/src/pages/home/aiFlows/ProjectsGallery.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { ProjectObjectType, SectionType, typedObjectImage } from '~/concepts/design/utils';
+import DividedGallery from '~/concepts/design/DividedGallery';
+import InfoGalleryItem from '~/concepts/design/InfoGalleryItem';
+import { SupportedArea } from '~/concepts/areas';
+import useIsAreaAvailable from '~/concepts/areas/useIsAreaAvailable';
+import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
+
+const ProjectsGallery: React.FC<{ onClose: () => void }> = ({ onClose }) => {
+  const { status: pipelinesAvailable } = useIsAreaAvailable(SupportedArea.DS_PIPELINES);
+  const servingPlatformStatuses = useServingPlatformStatuses();
+  const modelMeshEnabled = servingPlatformStatuses.modelMesh.enabled;
+
+  const projectDescription = `Projects allow you to organize your related work in one place. You can add workbenches, ${
+    pipelinesAvailable ? 'pipelines, ' : ''
+  }cluster storage, data connections, and model${
+    modelMeshEnabled ? ' servers' : 's'
+  } to your project.`;
+
+  return (
+    <DividedGallery
+      minSize="225px"
+      itemCount={3}
```

Comment:
I get it is static, but would be awesome if this was also a `.length` operation.